src/cli.ts

```
@@ -140,72 +105,53 @@ async function parseArguments(): Promise<Parsed> {
       },
     })
-    .option('webhook-url', {
+    .option('webhook-url', { string: true })
+    .option('admin-port', { number: true, demandOption: true })
+    .option('tenancy', { boolean: true, default: false })
+    .option('did-registry-contract-address', { string: true })
+    .option('schema-manager-contract-address', { string: true })
+    .option('wallet-connect-timeout', { number: true })
+    .option('wallet-max-connections', { number: true })
+    .option('wallet-idle-timeout', { number: true })
+    .option('apiKey', {
```

ESLint

[error] Insert `··`

(prettier/prettier)